### PR TITLE
Refactori18n

### PR DESCRIPTION
**Pull request by DaviRyan**

### CODE REVIEW

**Comment by markewallace** [comment]
One comment on the name LocaleUtil. By using uppercase for the first letter this indicates the module is a class but it's actually just a collection of methods. The normal convention is to use a lowercase first letter.

**Comment by DaviRyan** [comment]
Will change that and re deliver , will also investigate travis build failure.

**Comment by DaviRyan** [comment]
Name change delivered, Travis build failed previously because of test failures in the sbt.core.test project

**Comment by prb112** [comment]
yeah - I think we need to have a facade jar file in the test project for javax/mail/BodyPart, it'll enable the Travis Build to work.

File: sdk/com.ibm.sbt.web/src/main/webapp/js/sdk/_bridges/dojo-amd/i18n.js [hunk 1]
```diff
@@ -17,22 +17,10 @@
 /**
  * Social Business Toolkit SDK.
  */
-define(['dojo/i18n', 'dojo/date/locale'],function(i18n, dateLocale) {
+define(['dojo/i18n'],function(i18n) {
 	    var load = function(id, require, callback){	    	
 	    	i18n.load(id, require, callback); 
 	    };
```
Comment (ManishKataria) [inline sdk/com.ibm.sbt.web/src/main/webapp/js/sdk/_bridges/dojo-amd/i18n.js]: Should we be still returning load object from here ?

Comment (projsaha) [inline sdk/com.ibm.sbt.web/src/main/webapp/js/sdk/_bridges/dojo-amd/i18n.js]: Few Comments - 
1. Shouldnt we need to make similar changes in the _bridges/dojo/i18n module to remove the extra dependency on dojo/date/locale ? Also in that file we can remove dojo,require statements when we have the amdcompat js. 
2. ConnectionsGridRenderer and TemplatedGridRenderer also has reference to i18n.getUpdatedLabel. They also need to be changed

**Comment by projsaha** [comment]
Looks good to me. Thanks !

**Comment by DaviRyan** [comment]
@markewallace Do I need to manually make changes to the osgi package or is this done by the build.

**Comment by markewallace** [comment]
@DaviRyan normally you shouldn't have to change osgi. But can you explain what you need?

**Comment by DaviRyan** [comment]
@markewallace I have made all of the required changes but their is a comment from Projjwal above asking if we need to make the changes to the osgi modules also.

**Comment by markewallace** [comment]
@DaviRyan there is (or should be) only one copy of the JS files in the project. The build is responsible for copying the files so we can build the war and osgi targets. The version of the JS file in the com.ibm.sbt.web project is the master copy.

**Comment by prb112** [comment]
@projsaha or @ManishKataria could you merge the pull request?

**Comment by prb112** [comment]
@projsaha and @ManishKataria I'll merge on Thursday, unless I learn otherwise.

**Pull request merged**